frontend/toolkit-webapp/src/pages/ForgotPassword.js

```diff
@@ -1,8 +1,81 @@
-function ForgotPassword() {
+import React, { useState } from 'react'
+import { Button, Container, TextField, makeStyles, Typography, Paper, Box } from '@material-ui/core'
+import Avatar from '@material-ui/core/Avatar'
+import PermIdentityOutlinedIcon from '@material-ui/icons/PermIdentityOutlined';
+import config from '../config.json'
+import resetPassStyle from '../styles/registerStyle'
+// import '../css/Registration.css';
+import TopNavBar from '../components/TopNavBar'
+import videoSource from '../img/PEOSTRI.mp4'
+
+const ForgotPassword = (props) => {
+    
+    const [email, setEmail] = useState('')
+    const classes = resetPassStyle()
+
+    const onLogin = (e) => {
+        props.history.push('login');
+    }
+
+    const resetLink = async (e) => {
+        e.preventDefault()
+
+        const res = await fetch(config.server_url + config.paths.forgotPassword, {
+            method: 'POST',
+            headers: {
+                'Content-type': 'application/json'
+            },
+            body: JSON.stringify({ "email": email })
+        })
+
+        const data = await res.json()
+        alert('An email has been sent to the email listed, if registered.')
+
+    }
+
     return (
-        <div>
-            This is the forgot password page.
-        </div>
+        <>
+            <TopNavBar hideComponents={true}/>
```

Comment:
Home button not hidden on landing page. Clicking it causes dashboard to be routed with blank pages.

Reply:
Fixed this!